src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -289,6 +289,27 @@ module "pgbouncer_nlb__{{ pgbouncer_nlb.name }}" {
 
 {% endfor %}
 
+{% for proxy_alb in proxy_albs %}
+module "proxy_alb__{{ proxy_alb.name }}" {
+  source = "./modules/proxy_alb"
```

Comment:
Like my comment above, I think we want to call the module either `couch_alb` or `internal_alb` depending on the direction selected above